install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,279 @@ by performing the following steps:
 $ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/byo/openshift-cluster/openshift-metrics.yml \
    -e openshift_metrics_install_metrics=False
 ----
+
+[[openshift-prometheus]]
+== Prometheus on {product-title}
+
+Prometheus is a stand-alone, open source systems monitoring and alerting
+toolkit. You can use Prometheus to visualize metrics and alerts for {product-title}
+system resources.
+
+[IMPORTANT]
+====
+Prometheus on OpenShift is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[[openshift-prometheus-roles]]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* The `openshift-metrics` namespace.
+* Prometheus `clusterrolebinding` and service account.
+* Prometheus pod with Prometheus behind OAuth proxy, Alertmanager, and
+Alert Buffer as a stateful set.
+* Prometheus and `prometheus-alerts` ConfigMaps.
+* Prometheus and Prometheus Alerts services and direct routes.
+
+Prometheus deployment is disabled by default, enable it by setting
+`openshift_hosted_prometheus_deploy` to `true`. For example:
+
+----
+# openshift_hosted_prometheus_deploy=true
+----
+
+Set the following role variables to install and configure Prometheus.
+
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_namespace`
+|Project namespace where the components are deployed. Default set to
+`openshift-metrics`. For example, `openshift_prometheus_namespace=${USER_PROJECT}`.
+
+|`openshift_prometheus_node_selector`
+|Selector for the nodes on which Prometheus is deployed.
+
+|`openshift_prometheus_storage_kind`
```

Comment:
@zgalor 
Do we have the following 3 parameters? I did not find them in our openshift-ansible playbooks
openshift_prometheus_storage_kind
openshift_prometheus_alertmanager_storage_kind
openshift_prometheus_alertbuffer_storage_kind

Reply:
yes. you can see them in:
https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L643